tests/testthat/test-retriever.R

```diff
@@ -1,7 +1,147 @@
-library(testthat)
-library(rdataretriever)
+context("regression tests")
+
+library(DBI)
+library(RPostgreSQL)
+library(RMySQL)
+library(RSQLite)
+
 
 test_that("datasets returns some known values", {
   skip_on_cran()
   expect_identical("car-eval" %in% rdataretriever::datasets(), TRUE)
 })
+
+
+test_that("Download the raw portal dataset into './data/'", {
+  portal <- list("3299474", "3299483", "5603981")
+  rdataretriever::download('portal', './data/')
+  for (file in portal)
+  {
+    file_path <-
+      normalizePath(
+        file.path(getwd(), "tests/testthat/data", file),
+        winslash = "//",
+        mustWork = FALSE
+      )
+    expect_identical(identical(file.info(file_path)$size, integer(0)), FALSE)
+  }
+})
+
+
+test_that("Install the portal into csv", {
+  # Install portal into csv files in your working directory
+  portal <- list("portal_main", "portal_plots", "portal_species")
+  # rdataretriever::install('portal', 'csv')
+  for (file in portal)
+  {
+    file_path <-
+      normalizePath(
+        file.path(getwd(), "tests/testthat", paste(file, "csv", sep = ".")),
+        winslash = "//",
+        mustWork = FALSE
+      )
+
+    expect_identical(identical(file.info(file_path)$size, integer(0)), FALSE)
+  }
+})
+
+
+test_that("Install the portal into json", {
```

Comment:
Either drop the `the` or add `dataset` after Portal. The same change should be  made above and below.